IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -147,8 +148,8 @@ - (nullable MSIDCredentialCacheItem *)getCredential:(nonnull MSIDDefaultCredenti
 {
     assert(cacheQuery);
 
-    MSID_LOG_VERBOSE(context, @"(Default cache) Get accounts with environment %@", cacheQuery.environment);
-    MSID_LOG_VERBOSE_PII(context, @"(Default cache) Get accounts with environment %@, unique user id %@", cacheQuery.environment, cacheQuery.homeAccountId);
+    MSID_LOG_VERBOSE(context, @"(Default account cache) Get accounts with environment %@, hashed homeAccountId %@, isExactMatch %@, environment %@, environmentAliases %@", cacheQuery.environment, [cacheQuery.homeAccountId msidTokenHash], cacheQuery.exactMatch?@"yes":@"no" , cacheQuery.environment, cacheQuery.environmentAliases);
```

Comment:
Doesn't cacheQuery inherit from cacheItem? if yes, you can use hashedHomeAccountId instead of [cacheQuery.homeAccountId msidTokenHash]

Reply:
Hmm I checked and these 2 classes are from different families.